click/.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.idea/
```

Comment:
you need to remove .idea` now from the project to make it work
This folder is hidden, so you need to show it somehow on windows